Sources/TuistGenerator/Mappers/ModuleMapMapper.swift

```diff
@@ -2,6 +2,37 @@ import Foundation
 import TSCBasic
 import TuistCore
 import TuistGraph
+import TuistSupport
+
+enum ModuleMapMapperError: FatalError {
+    case invalidTargetDependency(sourceProject: AbsolutePath, sourceTarget: String, dependentTarget: String)
+    case invalidProjectTargetDependency(sourceProject: AbsolutePath, sourceTarget: String, dependentProject: AbsolutePath, dependentTarget: String)
+
+    /// Error type.
+    var type: ErrorType {
+        switch self {
+        case .invalidTargetDependency, .invalidProjectTargetDependency: return .abort
```

Comment:
is this misconfiguration on the user side? If it's due to how we handle SPM dependencies from `Dependencies.swift`, I'd mark this as `bug`.

Reply:
It can be both, so I would go with abort